entities/entities/spawned_money/cl_init.lua

```diff
@@ -3,27 +3,25 @@ include("shared.lua")
 function ENT:Initialize()
 end
 
+local vector_up = Vector(0,0,1)
```

Comment:
- Not the same as it was before (`Pos + Ang:Up() * 0.9`)
- Doesn't make it more modifiable